native_client/deepspeech.cc

```diff
@@ -56,13 +59,39 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
     return;
   }
 
-  Status status = NewSession(SessionOptions(), &mPriv->session);
+  Status status;
+  SessionOptions options;
+  bool is_mmap = std::string(aModelPath).find(".pbmm") != std::string::npos;
+  if (!is_mmap) {
+    std::cerr << "Warning: this model not mmap(). It will consume much more heap memory." << std::endl;
```

Comment:
Maybe rephrase this message as "Warning: reading entire model file into memory. Transform model file into an mmapped graph to reduce heap usage."